schema.sql

```diff
@@ -9,3 +9,27 @@ CREATE TABLE animals (
 	weight_kg  decimal,
 	species varchar(50)
 );
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(255),
+    age INTEGER
+);
+
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
+
+
+ALTER TABLE animals
+DROP COLUMN species;
+
+ALTER TABLE animals
```

Comment:
- [ ] Good job adding the required columns and doing most of the highlighted changes by @ahmetbozaci, But I notice you have missed one of the requirements, which is to set the id as autoincremented PRIMARY KEY for the animal table. Please remember we are required to `Make sure that id is set as autoincremented PRIMARY KEY`.